engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java

```diff
@@ -1335,15 +1125,11 @@ private void validatePropertyForNonDefaultGroup(ValidationContext<?> validationC
 	 *
 	 * @param validationContext The validation context.
 	 * @param propertyPath The property path for which constraints have to be collected.
-	 * @param metaConstraints A list where {@code MetaConstraint}s which match the given path are saved.
-	 * @param typeArgumentConstraints A list where type arguments {@code MetaConstraint}s which match the given path are saved.
 	 *
 	 * @return Returns an instance of {@code ValueContext} which describes the local validation context associated to the given property path.
 	 */
 	private <V> ValueContext<?, V> collectMetaConstraintsForPathWithValue(ValidationContext<?> validationContext,
```

Comment:
We should change the name of the method as it's not collecting constraints anymore.